portfolio/src/main/webapp/blog_script.js

```diff
@@ -51,6 +64,33 @@ function deleteComment(comment){
     fetch('/delete-comment', {method: 'POST', body:params});
 }
 
+function  getPostFunctionality(){
+    fetch('/login').then(response => response.json()).then((userInfo) =>{
+        if(userInfo[0] == "true"){
+            console.log('User is signed in and should see post option');
+            var forms = document.getElementsByTagName("form");
+            for(let i=0;i<forms.length; i++){
+               forms[i].style.display="block";
+            }
+        }else{
+         console.log('user not login and should not see post option');
+        }
+    })
+}
+
+function getUserLogin() {
+    fetch('/login').then(response => response.json()).then((userInfo) => {
+        const link = document.getElementById('loginButton');
+        link.href = userInfo[3];
+        if (userInfo[0] == "true") {
+            link.innerText="Sign Out";
+        }else{
```

Comment:
Could you apply formatter please. (Spaces between if/else)

Reply:
done.